internal/cocoasdk/api_coregraphics_darwin.go

```diff
@@ -0,0 +1,5 @@
+package cocoasdk
+
+type CGSize struct {
+	Width, Height float64
```

Comment:
Isn't this float32?

Reply:
CGSize is made up of CGFloat which is a double. [See Apple docs](https://developer.apple.com/documentation/corefoundation/cgfloat?language=objc)

Reply:
Acknowledged, thanks!